pkg/virt-handler/device-manager/device_controller_test.go

```diff
@@ -73,43 +71,6 @@ var _ = Describe("Device Controller", func() {
 			res = deviceController.nodeHasDevice(devicePath)
 			Expect(res).To(BeTrue())
 		})
-
```

Comment:
can we get a unit test that exercise the retry/re-register logic now that these "blocking" unit tests are removed?

Reply:
Done